src/acquisition/covidcast/csv_importer.py

```diff
@@ -223,6 +223,11 @@ def extract_and_check_row(row, geo_type):
       if len(geo_id) != 2 or not 'aa' <= geo_id <= 'zz':
         return (None, 'geo_id')
 
+    elif geo_type == 'nation':
+      # geo_id is lowercase
+      if not 'a' <= geo_id <= 'z':
```

Comment:
would we want this in 'aa' -> 'zz' (im presuming using standard international country codes (eg: us, ca, mx, gb, etc)

Reply:
good idea, done